doc/_config.yml

```diff
@@ -18,6 +18,7 @@ sphinx:
       navigation_with_keys: true
     html_static_path:
       - static/opengraph
+    myst_heading_anchors: 2
```

Comment:
what is this doing? why do we need it? please add a comment so we know why we added it